### PR TITLE
Add human-readable schemas

### PR DESCRIPTION
**Pull request by shaobo-he-aws**
*Issue #, if available:*
#102 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aaronjeline — APPROVED** [review]
Any reason to keep the json schemas?

**Comment by shaobo-he-aws** [comment]
> Any reason to keep the json schemas?

Since we support two kinds of schema formats, it doesn't hurt to keep them.

**Comment by cdisselkoen** [comment]
> Since we support two kinds of schema formats, it doesn't hurt to keep them.

By this rationale, we should add the JSON versions of every policy in the examples.  Is that what we want?

**Comment by shaobo-he-aws** [comment]
> > Since we support two kinds of schema formats, it doesn't hurt to keep them.
> 
> By this rationale, we should add the JSON versions of every policy in the examples. Is that what we want?

I think my argument is in favor of keeping JSON versions but not adding them.

**Comment by khieta** [comment]
I'm also in favor of deleting the JOSN schemas. Users can recreate them easily using the Cedar CLI.

**Comment by shaobo-he-aws** [comment]
> I'm also in favor of deleting the JOSN schemas. Users can recreate them easily using the Cedar CLI.

Good point. Updating.

File: cedar-example-use-cases/github_example/github_example.cedarschema [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  resource: [Repository]
+};
+action "edit_issue" appliesTo {
+  principal: [User],
+  resource: [Issue]
+};
+action "assign_issue" appliesTo {
+  principal: [User],
+  resource: [Issue]
+};
```
[the remark this answers is not in the document]
Comment (shaobo-he-aws) [reply cedar-example-use-cases/github_example/github_example.cedarschema]: Done.

**Review by john-h-kastner-aws — APPROVED** [review]
I also prefer deleting the JSON for the examples.

Although, we should be sure to keep sufficient JSON test case around. There aren't that many test cases _explicitly_ testing the json schema format, but lots of test cases incidentally testing it as part of typechecking tests.

**Review by khieta — APPROVED** [review]

**Comment by khieta** [comment]
Could you cherry-pick this to `main` too?

**Comment by mwhicks1** [comment]
> we should be sure to keep sufficient JSON test case around.

Are we testing the tool that converts natural schemas to JSON ones, and confirms that they behave the same?

**Comment by khieta** [comment]
> Are we testing the tool that converts natural schemas to JSON ones, and confirms that they behave the same?

We have fuzzing targets checking that the converter correctly round trips schemas (https://github.com/cedar-policy/cedar-spec/blob/main/cedar-drt/fuzz/fuzz_targets/json-schema-roundtrip.rs). But we don't have any tests that check that converted schemas produce the same validation results. If you think that would be useful, feel free to file an issue in [cedar-spec](https://github.com/cedar-policy/cedar-spec/issues).

**Pull request merged**